tests/models/textnet/test_modeling_textnet.py

```diff
@@ -0,0 +1,356 @@
+# coding=utf-8
+# Copyright 2023 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Testing suite for the PyTorch TextNet model. """
+import inspect
+import unittest
+
+import requests
+from PIL import Image
+
+from transformers import TextNetConfig
+from transformers.models.textnet.image_processing_textnet import TextNetImageProcessor
+from transformers.testing_utils import (
+    require_torch,
+    require_vision,
+    slow,
+    torch_device,
+)
+from transformers.utils import is_torch_available
+
+from ...test_backbone_common import BackboneTesterMixin
+from ...test_configuration_common import ConfigTester
+from ...test_modeling_common import ModelTesterMixin, floats_tensor, ids_tensor
+from ...test_pipeline_mixin import PipelineTesterMixin
+
+
+if is_torch_available():
+    import torch
+
+    from transformers import (
+        TextNetBackbone,
+        TextNetForImageClassification,
+        TextNetModel,
+        is_torch_available,
+    )
+    from transformers.models.textnet.modeling_textnet import TEXTNET_PRETRAINED_MODEL_ARCHIVE_LIST
+
+
+class TextNetModelTester:
+    def __init__(
+        self,
+        parent,
+        stem_kernel_size=3,
+        stem_stride=2,
+        stem_in_channels=3,
+        stem_out_channels=64,
+        # use_bn=True,
+        stem_act_func="relu",
+        dropout_rate=0,
+        ops_order="weight_bn_act",
+        conv_layer_kernel_sizes=[
+            [
+                [3, 3],
+            ],
+            [
+                [3, 3],
+            ],
+            [
+                [3, 3],
+            ],
+            [
+                [3, 3],
+            ],
+        ],
+        conv_layer_strides=[
+            [
+                2,
+            ],
+            [
+                2,
+            ],
+            [
+                2,
+            ],
+            [
+                2,
+            ],
+        ],
```

Comment:
nit - we don't need to use so much space here 

```suggestion
        conv_layer_kernel_sizes=[
            [[3, 3]],
            [[3, 3]],
            [[3, 3]],
            [[3, 3]],
        ],
        conv_layer_strides=[[2], [2], [2], [2]],
```

Reply:
Done.